public/app/features/datasources/components/tests/ButtonRow.test.tsx

```diff
@@ -28,7 +20,7 @@ const setup = (propOverrides?: object) => {
   return render(<ButtonRow {...props} />);
 };
 
-describe('Button Row', () => {
+describe('<ButtonRow>', () => {
```

Comment:
What is our best practice layout?  I have been adding tests to the *same* folder as the component, not a parallel test folder.

Should these tests all be in the components folder?

Reply:
Sorry, answered it [here](https://github.com/grafana/grafana/pull/51874#issuecomment-1188661685).

Reply:
Fixed it in https://github.com/grafana/grafana/pull/51874/commits/256d656b12bec45319425512f172ec8d171e1f07.